review-me/review-me.js

```diff
@@ -0,0 +1,26 @@
+// hello.js
+
+var message = "Hello, world!"
```

Comment:
3  :  MESSAGE  is  not  defined  Reference Error  at  runtime  because  Java Script  is  case -sensitive  (' message '  vs  ' MESSAGE ').